### ISSUE TITLE
Play Billing library version 3 ?

### ISSUE THREAD

**Issue by Gokulv617** [myflashlab/ANE-LAB#156]
I'm getting the following warning for all my apps in google playstore. I'm using the latest ANE's from ANELAB. May I know when  this update is going to happen ?

### Play Billing library version Deprecation
![img_20210324](https://user-images.githubusercontent.com/43432782/112254234-453c0600-8c86-11eb-837e-bc253e5a94a7.png)

**Comment by myflashlab** [comment]
Yes, we are aware of that and we will update the billing ANE before it’s too late of course.

**Comment by Gokulv617** [comment]
_**Thank you**_

**Comment by Gokulv617** [comment]
is this implemented in the new version of Billing ANE v5.0.0 ?
